.prettierignore

```diff
@@ -5,3 +5,4 @@
 **/.docusaurus/**
 **/dist/**
 **/relay-crates/**
+isograph-config-schema.json
```

Comment:
Is there a reason we should ignore the json file? It seems like we should always check in the latest version, just like we ensure that the demo artifacts are current (etc)

Wdyt?

Reply:
This is prettier ignore, we don't format any of the artifacts.

Reply:
oops, you're right

Reply:
So I guess, why are we uploading the artifact (on linux)?

The way I see it, we have the artifact checked in, so instead of uploading and downloading it, we can just use the checked in version?

Reply:
To check if it's been built before check-in